torch/_inductor/fx_passes/reinplace.py

```diff
@@ -754,5 +754,6 @@ def tensor_with_same_storage_already_reinplaced(arg):
 def reinplace_inplaceable_ops(graph: torch.fx.Graph) -> None:
     with enable_python_dispatcher():
         canonicalize_view_scatter_ops(graph)
+        torch._inductor.fx_passes.post_grad.fake_tensor_updater.incremental_update()
```

Comment:
The problem was:
- canonicalize_view_scatter_ops does some find and replacing of nodes.
- those new nodes have correct .meta[vals], but the users of said nodes have incorrect .meta[vals]
- so we need to update said users via FakeTensorUpdater.

Question: I don't know how to pass the FakeTensorUpdater into this pass (the GraphTransformObserver stuff is difficult to work with) so I set it as a global variable. I assume that's not what we want.